services/server/src/handlers/lesson.ts

```diff
@@ -33,6 +39,18 @@ const createLessonPayload = zod.object({
   duration,
 });
 
+const updateLessonPayload = zod.object({
+  // NOTE: tutorId is used for notification system, therefore it
+  // should be supplied by the Client in favor of relieving the
+  // burden of dispatching more db requests from the server.
+  // (this can be solved by storing tutorId in lessons table)
+  tutorId: id,
+  lessonId: id,
```

Comment:
You can get the tutor id by quering for the lesson members. This is something we already have to do to verify that the student owns (or a member) of the lesson he is trying to update.